frontend/src/component/common/Notifications/Notifications.tsx

```diff
@@ -43,7 +42,7 @@ const StyledTypography = styled(Typography)(({ theme }) => ({
 }));
 
 const StyledPaper = styled(Paper)(({ theme }) => ({
-    minWidth: '400px',
+    minWidth: '420px',
```

Comment:
```suggestion
    minWidth: theme.spacing(52.5),
```

Reply:
I would say to not use theme.spacing at this point for big elements, its much easier to control when you know the size directly
- maybe later on will make sense after we will have a better structure